pkg/controller/util_test.go

```diff
@@ -14,881 +17,149 @@ import (
-			want: false,
-		},
-		{
-			name: "pvc does not have any labels",
-			args: args{pvcNoLbl, CDILabelKey, CDILabelValue},
-			want: false,
-		},
-		{
-			name: "pvc does have labels but pass in empty search strings",
-			args: args{pvc, "", ""},
-			want: false,
-		},
-	}
-	// checkIfLabelExists expects both label to be present and correct value to match
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			if got := checkIfLabelExists(tt.args.pvc, tt.args.lbl, tt.args.val); got != tt.want {
-				t.Errorf("checkIfLabelExists() = %v, want %v", got, tt.want)
-			}
-		})
-	}
-}
-
-func TestMakeCDIConfigSpec(t *testing.T) {
-	type args struct {
-		name string
-	}
-	config := createCDIConfigWithStorageClass("testConfig", "")
-
-	tests := []struct {
-		name          string
-		args          args
-		wantCDIConfig *cdiv1.CDIConfig
-	}{
-		{
-			name:          "expect CDIConfig to be created",
-			args:          args{"testConfig"},
-			wantCDIConfig: config,
-		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			got := MakeEmptyCDIConfigSpec(tt.args.name)
-
-			if !reflect.DeepEqual(got, tt.wantCDIConfig) {
-				t.Errorf("MakeEmptyCDIConfigSpec() =\n%v\n, want\n%v", got, tt.wantCDIConfig)
-			}
-
-		})
-	}
-}
-
-func Test_addToMap(t *testing.T) {
-	type args struct {
-		m1 map[string]string
-		m2 map[string]string
-	}
-	tests := []struct {
-		name string
-		args args
-		want map[string]string
-	}{
-		{
-			name: "use different key for map1 and map2 expect both maps to be returned",
-			args: args{map[string]string{AnnImportPod: "mypod"}, map[string]string{CDILabelKey: CDILabelValue}},
-			want: map[string]string{AnnImportPod: "mypod", CDILabelKey: CDILabelValue},
-		},
-		{
-			name: "use same key for map1 and map2 expect map2 to be returned",
-			args: args{map[string]string{AnnImportPod: "mypod"}, map[string]string{AnnImportPod: "map2pod"}},
-			want: map[string]string{AnnImportPod: "map2pod"},
-		},
-		{
-			name: "pass in empty map1 and map2 expect empty map",
-			args: args{nil, nil},
-			want: map[string]string{},
-		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			if got := addToMap(tt.args.m1, tt.args.m2); !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("addToMap() = %v, want %v", got, tt.want)
-			}
-		})
-	}
-}
-
-func Test_getCertConfigMap(t *testing.T) {
-	namespace := "default"
-	configMapName := "foobar"
-
-	testPVC := &corev1.PersistentVolumeClaim{
-		ObjectMeta: metav1.ObjectMeta{
-			Annotations: map[string]string{
-				"cdi.kubevirt.io/storage.import.certConfigMap": configMapName,
-			},
-		},
-	}
-
-	testConfigMap := &corev1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      configMapName,
-			Namespace: namespace,
-		},
-	}
-
-	type args struct {
-		pvc    *corev1.PersistentVolumeClaim
-		objs   []runtime.Object
-		result string
-	}
-
-	for _, arg := range []args{
-		{&corev1.PersistentVolumeClaim{}, nil, ""},
```

Comment:
In the interest of completeness, it looks like this case was dropped in the new getCertConfigMap tests (2 cases vs. 3).

Reply:
You are right, its not a very interesting case, but we should keep it for completeness.